File: node-server/publicationVolume.js
const {CreativeWork} = require("./creativeWork.js");

class PublicationVolume extends CreativeWork{
	update(idPublicationVolume, alternativeHeadline, commentCount, copyrightYear, inLanguage, isAccessibleForFree, pageStart, pageEnd, pagination, volumeNumber){
		super.update(idPublicationVolume, alternativeHeadline, commentCount, copyrightYear, inLanguage, isAccessibleForFree);
        this.pageStart = pageStart;
        this.pageEnd = pageEnd;
		this.pagination = pagination;
		this.volumeNumber = volumeNumber;
	}
}

module.exports = {PublicationVolume};

File: python-server/application/model/publicationVolume.py
from application.model.creativeWork import CreativeWork

__author__ = 'Lucía Blanco Rodríguez'


class PublicationVolume(CreativeWork):
    def __init__(self, id, alternativeHeadline, commentCount, copyrightYear, inLanguage, isAccessibleForFree, pageStart, pageEnd, pagination, volumeNumber):
        super().__init__(id, alternativeHeadline, commentCount, copyrightYear, inLanguage, isAccessibleForFree)
        self.pageStart = pageStart
        self.pageEnd = pageEnd
        self.pagination = pagination
        self.volumeNumber = volumeNumber
